subsys/bluetooth/host/att.c

```diff
@@ -194,7 +195,11 @@ static inline void tx_meta_data_free(struct bt_att_tx_meta_data *data)
 	__ASSERT_NO_MSG(data);
 
 	(void)memset(data, 0, sizeof(*data));
-	k_fifo_put(&free_att_tx_meta_data, data);
+
```

Comment:
if (data == chan.rsp_data) { /* "Freeness" is kept track of by remote (you can only have one transaction). */ }
else {assert(part_of_array())}